recipes/source.rb

```diff
@@ -39,7 +39,7 @@
   source "#{node['php']['url']}/php-#{version}.tar.gz/from/this/mirror"
   checksum node['php']['checksum']
   mode '0644'
-  not_if "which #{node['php']['bin']}"
+  not_if "which #{node['php']['bin']} --version | grep #{version}"
```

Comment:
**This is always false.** It should be something like this:
```ruby
not_if "$(which #{node['php']['bin']}) --version | grep #{version}"
```

It does not return PHP version but version of which command on CentOS and only PHP path on Ubuntu

CentOS 7:
```
[root@test ~]# which php --version
GNU which v2.20, Copyright (C) 1999 - 2008 Carlo Wood.
GNU which comes with ABSOLUTELY NO WARRANTY;
This program is free software; your freedom to use, change
and distribute this program is protected by the GPL.
```
Ubuntu 14.04:
```
[root@test ~]#which php --version
/usr/bin/php
```